### PR TITLE
Feat/add container insights

### PR DESCRIPTION
**Pull request by klebertertulino**
- Update CDK versions
- Silence deprecated warnings
- Update Node NVM and NPM versions
- Fix vault to run locally
- Add container insights to ECS cluster
- Standardize the ops.yml versions

### CODE REVIEW

File: src/vault.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ async function destroy() {
 
 }
 
-switch(ARGS[0]) {
```
[the remark this answers is not in the document]
Comment (kc-dot-io) [reply src/vault.ts:216]: @klebertertulino Can you confirm that it runs both locally and when run from remote?

The main issue is that when you `ops run -b . ls` the ARGS is +1 vs `ops run vault ls` or `ops run . ls`.

So please make sure each scenarios work so we don't solve one and break the other.

[the remark this answers is not in the document]
Comment (kc-dot-io) [reply src/vault.ts:216]: Also I would suggest breaking this out into a different PR so as to not mix it with this feature. Thanks!

[the remark this answers is not in the document]
Comment (klebertertulino) [reply src/vault.ts]: Reverted that change.

**Comment by kc-dot-io** [comment]
@klebertertulino We're going to have to resolve some minor conflicts here once #28 merges.

**Pull request merged**